src/test/kotlin/MainKtTest.kt

```diff
@@ -1,10 +1,2 @@
-import org.junit.Test
-import kotlin.test.assertEquals
 
-internal class MainKtTest {
-
-    @Test
-    fun testGetSum() {
-        assertEquals(4, getSum(2, 2))
-    }
-}
\ No newline at end of file
+internal class MainKtTest
```

Comment:
Is it possible to create some simple tests? Which check filtering repositories by different fields.